app/templates/participate/login.hbs

```diff
@@ -24,7 +24,7 @@
     <div class="row">
       <div class="col-md-12">
         <h1>International Situations Project</h1>
-        {{isp-jam-auth login=(action 'authenticate')}}
+        {{isp-jam-auth invalidAuth=invalidAuth authenticating=authenticating login=(action 'authenticate')}}
```

Comment:
I can't seem to find where the `isp-jam-auth` component uses the `invalidAuth` variable- can you please clarify why this needs to be passed in?